include/gazebo_mavlink_interface.h

```diff
@@ -357,5 +383,7 @@ class GazeboMavlinkInterface : public ModelPlugin {
 
   bool hil_mode_;
   bool hil_state_level_;
+
+  std::atomic<bool> gotSigInt_ {false};
```

Comment:
```suggestion
  std::atomic<bool> got_sig_int_ {false};
```

Although I see we already have inconsistencies.